kubernetes-csi-external-attacher-4.4.yaml

```diff
@@ -1,10 +1,13 @@
 package:
-  name: kubernetes-csi-external-attacher
+  name: kubernetes-csi-external-attacher-4.4
   version: 4.4.0
   epoch: 0
   description: Sidecar container that watches Kubernetes VolumeAttachment objects and triggers ControllerPublish/Unpublish against a CSI endpoint
   copyright:
     - license: Apache-2.0
+  dependencies:
+    provides:
+      - kubernetes-csi-external-attacher=4.4.999
```

Comment:
Same as above, bump the epoch and use `${{package.full-version}}`

Reply:
Done, latest was 0, https://github.com/wolfi-dev/os/blob/main/kubernetes-csi-external-attacher.yaml